src/App.css

```diff
@@ -37,3 +40,77 @@
     transform: rotate(360deg);
   }
 }
+
+body {
+  background: linear-gradient(to right, #0af, #00ff6c);
+}
+
+header h2 {
+  text-align: center;
+}
+
+.calculator {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  background-color: gray;
+}
+
+.result {
+  display: grid;
+  float: right;
+  grid-column: 1 / -1;
+  margin-right: 4rem;
+  grid-template-columns: repeat(4, 5.5rem);
+  grid-template-rows: minmax(7rem, auto) repeat(5, 5rem);
+}
+
+.keypad,
+.display {
+  width: 80%;
+  height: 20px;
+  background-color: #9e9999;
+  border: 1px solid #ccc;
+  border-radius: 2px;
+  font-size: 16px;
+  cursor: pointer;
+  justify-content: center;
+  align-items: center;
+}
+
+.calculate .keypad {
+  display: grid;
+  grid-template-columns: repeat(4, 2fr);
+}
+
+.keypad .btn-Operator {
+  color: #333;
+  background-color: orange;
+  transition: 0.2s ease;
+}
+
+.display {
+  color: white;
+  text-align: right;
+  background-color: gray;
+  border: 1px solid grey;
+  transition: 0.2s ease;
+}
+
+.keypad button {
+  background-color: whitesmoke;
+}
+
+.keypad button:hover {
+  background-color: wheat;
+}
+
+.z {
+  grid-column-start: 1;
+  grid-column-end: 3;
+}
+
+.keypad .btn-Operator:hover {
+  background-color: orangered;
+}
```

Comment:
- [ ] kindly consider making sure that your design follows the design snippet you were given in the project requirement to make your design implementation an exact duplicate of the given one  check the images below for more clarrification
- your design 👍 
<img width="927" alt="Screen Shot 2023-06-03 at 08 47 16" src="https://github.com/supreeti/Math-magician-App/assets/60146030/ab00d292-7c03-4c15-babd-30f90a9e3da8">

- given design to implement 👇 
<img width="302" alt="Screen Shot 2023-06-03 at 08 51 59" src="https://github.com/supreeti/Math-magician-App/assets/60146030/1a5b37c1-24f5-4b74-bac9-b3b8c0bbef3f">